### PR TITLE
at-handle

### PR DESCRIPTION
**Pull request by tncarr101**

### CODE REVIEW

**Review by claesmk — APPROVED** [review]
I'm approving this as-is, however, if you noticed the problem with the space between the replacement text and the word before the at handle getting removed I started a [discussion](https://github.com/orgs/NCATComp410/teams/comp410spring2022class/discussions/16) about that. 

Please remember to have someone else on your team review your work then your pull request will merge.

**Comment by claesmk** [comment]
Reminder you are waiting for someone else on your team to review your changes - after that you will be able to merge the pull request.

**Review by kaylaqhenry — APPROVED** [review]

**Pull request merged**
